tests/containers/docker.pm

```diff
@@ -51,7 +54,12 @@ sub run {
     test_seccomp();
     allow_selected_insecure_registries(runtime => 'docker');
 
-    check_docker_firewall() if $self->firewall() eq 'firewalld';
+    if ($self->firewall() eq 'firewalld') {
+        # on publiccloud we need to install firewalld first
+        install_and_start_firewalld() if (is_publiccloud || is_jeos);
+        check_docker_firewall();
+        systemctl('stop firewalld') if $stop_firewall;
```

Comment:
with `post_fail_hook` and `post_run_hook` this line does not make sense

Reply:
Outdated.